packages/storybook/stories/va-link.stories.jsx

```diff
@@ -0,0 +1,159 @@
+import React, { Fragment } from 'react';
+import { getWebComponentDocs, propStructure, StoryDocs } from './wc-helpers';
+
+const linkDocs = getWebComponentDocs('va-link');
+
+export default {
+  title: 'Components/va-link',
+  parameters: {
+    componentSubtitle: `Link web component`,
+    docs: {
+      page: () => <StoryDocs data={linkDocs} />,
+    },
+  },
+};
+
+const defaultArgs = {
+  active: undefined,
+  channel: undefined,
+  download: undefined,
+  href: 'https://www.va.gov',
+  filename: undefined,
+  video: undefined,
+};
+
+const Template = ({ active, channel, download, href, filename, video }) => {
+  return (
+    <va-link
+      active={active}
+      channel={channel}
+      download={download}
+      href={href}
+      filename={filename}
+      video={video}
+    >
+      Find out if you qualify for this program and how to apply
+    </va-link>
+  );
+};
+
+export const Default = Template.bind(null);
+Default.args = {
+  ...defaultArgs,
+};
+Default.argTypes = propStructure(linkDocs);
+
+const ActiveTemplate = ({
+  active,
+  channel,
+  download,
+  href,
+  filename,
+  video,
+}) => {
+  return (
+    <va-link
+      active={active}
+      channel={channel}
+      download={download}
+      href={href}
+      filename={filename}
+      video={video}
+    >
+      Share your VA medical records
+    </va-link>
+  );
+};
+
+export const Active = ActiveTemplate.bind(null);
+Active.args = {
+  ...defaultArgs,
+  active: true,
+};
+
+const DownloadTemplate = ({
+  active,
+  channel,
+  download,
+  href,
+  filename,
+  video,
+}) => {
+  return (
+    <va-link
+      active={active}
+      channel={channel}
+      download={download}
+      href={href}
+      filename={filename}
+      video={video}
+    >
+      Download VA form 10-10EZ
+      <dfn>
+        (<abbr title="Portable Document Format">PDF</abbr>, 5 pages)
+      </dfn>
```

Comment:
Since this looks like a standard format, is there a way we can parse the file and extract the page count in order to keep things consistent and accurate? I can imagine if someone updates the PDF asset but not the link that this bit of content could get out of date and be misleading. And since we want that content there by default, automatically including it could make the component easier to use and more reliable.

![Link guidance on documentation with the structure "(PDF, page count)" highlighted](https://user-images.githubusercontent.com/2008881/178370846-5ab3ad70-0ecd-49cc-9d22-2f99414b4a89.png)